net/dhcp6/files/patch-dhcp6c.c

```diff
@@ -0,0 +1,30 @@
+--- dhcp6c.c.orig	2017-02-28 19:06:15 UTC
++++ dhcp6c.c
+@@ -521,6 +521,7 @@ process_signals()
+ 	if ((sig_flags & SIGF_HUP)) {
+ 		d_printf(LOG_INFO, FNAME, "restarting");
+ 		free_resources(NULL);
++		cfparse(conffile);
```

Comment:
@marjohn56 is this the SIGHUP change?

Reply:
It is...

Reply:
ok, I can pull this in for 18.1.6 directly, add the RAW stuff too, but as an off by default ports option. 18.1.6 will produce new images so we keep the changes minimal.

It would have been easier to review a single line a while ago. There is nothing against it. Burying it in a large patch set is definitely inducing delay. ;(

Reply:
I've asked Kev to do a usage write up on the RAW options, it's his 'France' thing and he should write the documentation for it.... I just added it and tested it.

Reply:
Now now.. you should know by now I rarely do things in a small way. :)

Reply:
Separate issue. Have a read of https://forum.opnsense.org/index.php?topic=7719.0 and see my reply's on this. I don't see how we can automatically add a gateway when there is no way of knowing on which interface the request arrived. Fine with one LAN, more than that and it appears not possible.

Reply:
so here's step one... slightly modified for log message :) https://github.com/opnsense/ports/commit/bc8d0afbc34